pkgs/development/interpreters/php/default.nix

```diff
@@ -227,7 +229,10 @@ let
       '';
 
       src = fetchurl {
-        url = "https://www.php.net/distributions/php-${version}.tar.bz2";
+        url = [
+          "https://downloads.php.net/~derick/php-${version}.tar.xz"
```

Comment:
Is this temporary before 7.4 stable is released?

Reply:
I do not know

Reply:
It is, they don't release the source tarballs in the standard location for betas or RCs.

Reply:
You could also use: `https://github.com/php/php-src/archive/php-${version}.tar.gz` 
https://github.com/php/php-src/releases